### PR TITLE
4.0

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

**Review by ash3rz — APPROVED** [review]
:shipit:

File: package.json [hunk 1]
```diff
@@ -65,15 +62,16 @@
         "@babel/plugin-syntax-dynamic-import": "^7.2.0",
         "@babel/preset-env": "^7.3.4",
         "@babel/preset-react": "^7.0.0",
-        "@material-ui/core": "^3.9.3",
-        "@material-ui/icons": "^3.0.2",
-        "@storybook/addon-actions": "^5.1.3",
+        "@material-ui/core": "^4.3.0",
+        "@material-ui/icons": "^4.2.1",
+        "@material-ui/lab": "^4.0.0-alpha.20",
```
Comment (psarando) [inline package.json:67]: Can the `@material-ui` libs be moved back into `peerDependencies` now?
It seems like that's the more appropriate place for them 🤔

File: src/components/rating/Rate.js [hunk 2]
```diff
@@ -47,23 +42,15 @@ class Rate extends Component {
...
 
-        const iconHovered = orange[500];
-        const iconNormal = grey[500];
-
-        //if user can delete, then she/he owns the rating so style it differently
-        const iconFilled = onDelete ? iconHovered : red[500];
-
         return (
-            <div>
+            <React.Fragment>
```
Comment (psarando) [inline src/components/rating/Rate.js:46]: 👍

**Review by psarando — APPROVED** [review]
LGTM

I had a question about `peerDependencies` now that you found a fix for the Hooks errors.
Otherwise :shipit: 👍

File: package.json (hunk 1 above)
Comment (sri2k1us) [reply package.json:67]: @psarando Unfortunately no. If we do that, `Can't resolve '@material-ui/lab/Rating' in '/Users/sriram/cyverse-de/ui-lib/src/components/rating'` error.

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**